doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc

```diff
@@ -3,7 +3,7 @@
 
 For more efficient searching across cluster nodes, install {DATAGRID} and configure it for the {PRODUCT} clustered environment.
 
-For information about {DATAGRID} advanced installation and configuration options and {DATAGRID} modules for {EAP}, see the https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html-single/red_hat_data_grid_user_guide/index#[_{DATAGRID} User Guide_].
+For information about {DATAGRID} advanced installation and configuration options and {DATAGRID} modules for {EAP}, see the https://access.redhat.com/documentation/en-us/red_hat_data_grid/{DATAGRID_VERSION}/html-single/red_hat_data_grid_user_guide/index#[_{DATAGRID} User Guide_].
```

Comment:
Note that this URL does not work. I checked the DG 8.0 doc library (https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.0) and there isn't a User Guide anymore. Find out what doc you should reference now.

Reply:
Doc to reference is https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.1/html/data_grid_server_guide/index .